### DISCUSSION TITLE
Passive highlight (non-link) behavior

### DISCUSSION THREAD

**Discussion by Gnurrf** [RyotaUshio/obsidian-pdf-plus#155] [Q&A]
Hi all,

at the moment I am a bit stumped by the behavior I observe when trying to do "passive" highlights in the displayed PDF, not due to links.

a) I select a range of text, then hit a highlight color button -> It says link copied, but no highlighting occurs.
b) I select a highlight color button, then a range of text -> no highlighting occurs

If I do a), and then paste the link in my Markdown pane, I _do_ get a highlight, but that appears to be the _backlink_ highlight due to the link being present in Markdown.

Question: How do I add passive (non-link) highlights to be displayed in the PDF? Do I have to "enable PDF editing" for that? Or is there a way that leaves the original PDF untouched?

And before I forget again: PDF++ is simply amazing, I expect to build my entire workflow around it mid-term.

**Comment by RyotaUshio** [comment]
Hi, currently there is no way to add a highlight without creating a corresponding link or writing it directly into PDF (this is an intentional design choice for future-proofness; you might want to check out the discussion in #16 ).
To add a highlight without creating a link, first enable PDF editing, and then turn on this "add highlights to file directly" icon.

<img width="250" alt="image" src="https://github.com/RyotaUshio/obsidian-pdf-plus/assets/72342591/e7519a5e-ff59-4596-bee0-3b81edf1ecd9">
